# Encoder averaged speed for one channel in a Phidgets High-Speed Encoder board
Header header
# Averaged (sliding window) speed estimation [ticks/s]
float64 speed
